boofuzz/sessions.py

```diff
@@ -280,6 +280,9 @@ class Session(pgraph.Graph):
         web_port (int):         Port for monitoring fuzzing campaign via a web browser. Default 26000.
         fuzz_data_logger (fuzz_logger.FuzzLogger): DEPRECATED. Use fuzz_loggers instead.
         fuzz_loggers (list of ifuzz_logger.IFuzzLogger): For saving test data and results.. Default Log to STDOUT.
+        fuzz_loggers_keep_num_log_cases (int): Number of most recent test cases to save to db in case of an error or
+                                               failure. Set to 0 to save after every test step (high disk I/O!)
+                                               Default 0
```

Comment:
Here's a rewording suggestion, emphasizing the point of this setting (minimizing disk usage), and changing the second sentence a bit to emphasize which cases are saved rather than how frequently.
> Minimize disk usage by only saving passing test cases if they are in the n test cases preceding a failure or error. Set to 0 to save every single test case (high disk I/O!). Default 0.

Reply:
I'm thinking about names... what about `fuzz_db_keep_only_n_pass_cases`?

Reply:
> 
> 
> I'm thinking about names... what about `fuzz_db_keep_only_n_pass_cases`?

That sounds more intuitive, will rename.